hub/modules/connections/connections.bal

```diff
@@ -57,3 +57,12 @@ public isolated function createMessageConsumer(websubhub:VerifiedSubscription me
     };
     return check new (config:KAFKA_BOOTSTRAP_NODE, consumerConfiguration);  
 }
+
+// Consumer which reads the health check
+kafka:ConsumerConfiguration healthConsumerConfig = {
+    groupId: "health--check-topics-group-" + config:CONSTRUCTED_SERVER_ID,
+    offsetReset: "earliest",
+    topics: [ config:HEALTH_CHECK_TOPIC ]
+};
+public final kafka:Consumer healthCheckConsumer = check new (config:KAFKA_BOOTSTRAP_NODE, healthConsumerConfig);
```

Comment:
Why are we creating this consumer, can we use one of the existing consumers?

Reply:
using just producer to check connectivity.

Reply:
Why is healthConsumerConfig section not removed ?

Reply:
removed